bioblend/galaxy/libraries/__init__.py

```diff
@@ -748,13 +748,13 @@ def set_library_permissions(
         :return: General information about the library
         """
         payload: Dict[str, List[str]] = {}
-        if access_in:
```

Comment:
Before this test failed for `access_in` being `None` or `[]`. Now only for `None`. Is this desired?

Reply:
I think [] make things unrestricted and is desired in create public data library.

Reply:
Can you add a test that shows this?

I'm also not entirely sure what you mean with "I want to make the library and dataset public". Has this not been possible before. Again, a test would be highly appreciated.